app/models/tour_list.rb

```diff
@@ -0,0 +1,39 @@
+class TourList
+  attr_reader :contents
+
+  def initialize(initial_contents)
+    @contents = initial_contents || Hash.new(0)
+    @contents.default = 0
+  end
+
+  def total_count
```

Comment:
This and count_of seem like weird methods for a tour... Do you add the same landmark/recording multiple times? Makes more sense to just use an array instead of a hash -- will greatly simplify this class.

Reply:
`total_count` and `count_of` are not methods for a tour but are being used for "cart" functionality. You are right about using an array but I used a hash because that is what was on my mind at the time. We can refactor it into an array. RED, GREEN, REFACTOR